lot/landmapper/templates/landmapper/privacy.html

```diff
@@ -0,0 +1,159 @@
+{% extends 'landmapper/account/account_base.html' %}
+{% load static %}
+{% load i18n %}
+{% load account %}
+
+{% block styles %}
+  {{ block.super }}
+  <link rel="stylesheet" href="{% static 'landmapper/css/auth.css' %}">
+  <link rel="stylesheet" href="{% static 'landmapper/css/legal.css' %}">
+{% endblock styles %}
+
+{% block main %}
+<div id="privacy" class="legal-container">
+    <h1>Privacy Policy</h1>
```

Comment:
Adding a note to remind us to add [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) and [CCPR](https://en.wikipedia.org/wiki/California_Consumer_Privacy_Act)